query/common/src/main/java/org/codice/ddf/admin/common/fields/common/FilePathField.java

```diff
@@ -20,7 +20,9 @@
 
   public static final String FIELD_TYPE_NAME = "FilePath";
 
-  // TODO: tbatie - 4/3/17 - Enforce file path must be relative to ddf home?
+  @SuppressWarnings("squid:S1135" /* Remove when TODO is completed */)
+  // TODO: tbatie - 4/3/17 - Enforce file path must be relative to ddf home? Remove sonar
```

Comment:
Is anyone using this class? I think we can delete it if not.

Reply:
No, it's not being used anywhere. I'll go ahead and remove it.